.travis.yml

```diff
@@ -109,12 +109,38 @@ jobs:
           branch: master
           repo: denoland/deno
         skip-cleanup: true
+    
+    - name: "cargo release linux arm64"
+      os: linux
+      dist: xenial
+      after_success:
+        - ls target/aarch64-unknown-linux-gnu/release
+        - gzip -c target/aarch64-unknown-linux-gnu/release/deno > target/release/deno_${TRAVIS_OS_NAME}_arm64.gz
+      script:
+      - rustup target add aarch64-unknown-linux-gnu
+      - sudo apt update
+      - sudo apt -yq --no-install-suggests --no-install-recommends install g++-5-aarch64-linux-gnu gcc-5-aarch64-linux-gnu g++-5-multilib libc6-arm64-cross
+      - build/linux/sysroot_scripts/install-sysroot.py --arch=arm64
+      - export DENO_BUILD_ARGS="target_cpu=\"arm64\" v8_target_cpu=\"arm64\""
+      - export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER="/usr/bin/aarch64-linux-gnu-gcc-5"
+      - export CC_aarch64_unknown_linux_gnu="/usr/bin/aarch64-linux-gnu-gcc-5"
+      - ./tools/setup.py
+      - cargo build -vv --target=aarch64-unknown-linux-gnu --release --locked
+      deploy:
```

Comment:
Remove the deploy section for now. Because the build isn’t tested - we shouldn’t release.